2011-03-26  Aurelien Jarno  <aurelien@aurel32.net>

        * sysdeps/unix/alpha/sysdep.h (PTR_MANGLE): Move out of ASSEMBLER 
	ifdef. 
	(PTR_DEMANGLE): Likewise.

---
 sysdeps/unix/alpha/sysdep.h |    7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

--- a/sysdeps/unix/alpha/sysdep.h
+++ b/sysdeps/unix/alpha/sysdep.h
@@ -154,6 +154,8 @@
 
 #else /* !ASSEMBLER */
 
+#include <stdint.h>
+
 /* In order to get __set_errno() definition in INLINE_SYSCALL.  */
 #include <errno.h>
 
@@ -334,11 +336,11 @@
 	_sc_ret = _sc_0, _sc_err = _sc_19;			\
 }
 
+#endif /* ASSEMBLER */
+
 /* Pointer mangling support.  Note that tls access is slow enough that
    we don't deoptimize things by placing the pointer check value there.  */
 
-#include <stdint.h>
-
 #if defined NOT_IN_libc && defined IS_IN_rtld
 # ifdef __ASSEMBLER__
 #  define PTR_MANGLE(dst, src, tmp)				\
@@ -382,4 +384,3 @@
 # endif
 #endif
 
-#endif /* ASSEMBLER */
